ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1985,6 +1985,26 @@ HP2SC:
   implements:
   - CONTROL
 
+DXASP2:
```

Comment:
Please explain this type. I am unclear what the sequence is or how this would be utilized. I am also unclear on what the fans are -- supply fans, discharge fans? The type appears to be (from the required fields) zone air temperature and fan status monitoring, and has nothing to do with DX cooling control.